_includes/assets/js/inline.js

```diff
@@ -7,3 +7,20 @@ if (window.netlifyIdentity) {
     }
   });
 }
+
+$(document).ready(function(){
+	// Open and close loop for main navigation on mobile.
+	$(".header-main__menu").click(function(){ // when user clicks on nav btn..
```

Comment:
Am I right in thinking that users would need JS to access the navigation? Do you think we could make it so JS hides the menu so it's still accessibile if JS isn't available or hasn't loaded yet? I found a good example here: https://www.a11ymatters.com/pattern/mobile-nav/#4--progressive-enhancement

Reply:
Do you mean, '...so CSS hides the menu so its still accessible..', not '..JS..' ? if so, then yes  I think we can build a solution in.

Reply:
I think we need to discuss this. The navigation is full screen, so from a progressive enhancement viewpoint, without js, this will be full screen which means although the nav will appear, the user wont be able to see any of the content 'behind' it.

Reply:
Yeah you're right. In that case we can either:
1. Make the full screen styling conditional on JS and have some ugly but usable styling for no JS  
2. Use a CSS only solution for no-JS https://css-tricks.com/off-canvas-menu-with-css-target/

Reply:
Implemented as discussed.